### PR TITLE
refactor: design updates for input, label, fieldset, radio, checkbox

### PR DESCRIPTION
**Pull request by melaniebmn**
# Summary | Résumé

Design updates for input, label, fieldset, radio, checkbox. It also includes the token package update to the latest version `1.12.2`.

[Zenhub ticket input](https://app.zenhub.com/workspaces/design-system-6100624a19f4cf000e46e458/issues/gh/cds-snc/design-gc-conception/716)

[Zenhub ticket checkbox](https://app.zenhub.com/workspaces/design-system-6100624a19f4cf000e46e458/issues/gh/cds-snc/design-gc-conception/662)

[Zenhub ticket radio](https://app.zenhub.com/workspaces/design-system-6100624a19f4cf000e46e458/issues/gh/cds-snc/design-gc-conception/661)

### CODE REVIEW

File: packages/web/src/components/gcds-checkbox/gcds-checkbox.css [hunk 1]
```diff
@@ -21,41 +21,48 @@
...
-    max-width: 90%;
+    position: relative;
+    max-width: var(--gcds-checkbox-max-width);
+    min-height: calc(
+      var(--gcds-checkbox-input-height-and-width) - var(--gcds-checkbox-padding)
+    );
     font: var(--gcds-checkbox-font);
     color: var(--gcds-checkbox-default-text);
     margin: var(--gcds-checkbox-margin) !important;
+    padding: var(--gcds-checkbox-padding) 0 0;
```
Comment (ethanWallace) [inline packages/web/src/components/gcds-checkbox/gcds-checkbox.css:32]: Should we include the additional `0 0` in the `--gcds-checkbox-padding` token?

File: packages/web/src/components/gcds-radio-group/gcds-radio-group.css [hunk 2]
```diff
@@ -22,41 +22,53 @@
...
-    max-width: 90%;
+    position: relative;
+    max-width: var(--gcds-radio-max-width);
+    min-height: calc(
+      var(--gcds-radio-input-height-and-width) - var(--gcds-radio-padding)
+    );
     font: var(--gcds-radio-font);
     color: var(--gcds-radio-default-text);
     margin: var(--gcds-radio-margin) !important;
+    padding: var(--gcds-radio-padding) 0 0;
```
Comment (ethanWallace) [inline packages/web/src/components/gcds-radio-group/gcds-radio-group.css:33]: Same question about the extra `0 0` from above.

**Review by ethanWallace — CHANGES_REQUESTED** [review]
Looking good. Just a few comments around the padding tokens with radio and checkbox

File: packages/web/src/components/gcds-input/gcds-input.css [hunk 3]
```diff
@@ -43,6 +43,16 @@
         border-color ease-in-out 0.15s,
         box-shadow ease-in-out 0.15s,
         outline ease-in-out 0.15s;
+
+      /* Hide number arrows */
```
Comment (daine) [inline packages/web/src/components/gcds-input/gcds-input.css:47]: Nice to have this comment - makes it clear and easy to understand, thanks!

File: packages/web/src/components/gcds-checkbox/gcds-checkbox.css (hunk 1 above)
Comment (melaniebmn) [reply packages/web/src/components/gcds-checkbox/gcds-checkbox.css:32]: I didn't include the `0 0` to use the padding token to calculate the correct min-height for the checkbox.

File: packages/web/src/components/gcds-radio-group/gcds-radio-group.css (hunk 2 above)
Comment (melaniebmn) [reply packages/web/src/components/gcds-radio-group/gcds-radio-group.css:33]: The same comment - used the same token in the min-height calculation on line 28 which wouldn't
t be possible if I included the `0 0`.

**Review by ethanWallace — APPROVED** [review]
LGTM

**Pull request merged**